api/catalog/api/utils/validate_images.py

```diff
@@ -18,6 +19,10 @@ def _get_cached_statuses(redis, image_urls):
     return [int(b.decode("utf-8")) if b is not None else None for b in cached_statuses]
 
 
+def _get_expiry(status, default):
+    return config(f"LINK_VALIDATION_CACHE_EXPIRY__{status}", default=default, cast=int)
```

Comment:
This might be bikeshedding but the `__` in the name looks weird.

Reply:
It's a not-uncommon separator for environment variables: https://github.com/WordPress/openverse-catalog/blob/f5de3e2a5f9c9da62f7e47e9c86b566e3e345123/env.template#L11

Reply:
It's new to the API. But it's cool 👍. Could you move this to `settings` (so that almost all env reading happens in one place) and also add it to the `env.template` file?

Reply:
Move the `_get_expiry` function?

Reply:
Oh, my bad. I didn't realise that the `default=` argument to `config` was set by the caller to `_get_expiry`. Please ignore my suggestion.